src/redux/actions/loginAction.js

```diff
@@ -0,0 +1,46 @@
+import axios from 'axios';
+import { FETCH_USERS_REQUEST, FETCH_USERS_SUCCESS, FETCH_USERS_FAILURE, CLOSE_MESSAGE } from '../types/loginTypes';
```

Comment:
@blaise82 Could you use meaningful action types? Are you really fetching users? consider my suggestions below:
```suggestion
import { LOGIN_REQUEST, LOGIN_SUCCESS, LOGIN_FAILURE, CLOSE_MESSAGE } from '../types/loginTypes';
```